### ISSUE TITLE
Publish v0.1.0

### ISSUE THREAD

**Issue by danclaytondev** [olivere/vite#18]
I think we should look at releasing this as a v0.x experimental module.

TODO:
- Tidy and improve documentation
- Add v0.1.0 git tag
- Publish to pkg.go.dev
- Update proxy.golang.org (go list)

Any thoughts on what we need to do to make this more useful?

**Comment by olivere** [comment]
Yes, we could start publishing tagged versions. I'm quite happy with [GoReleaser](https://goreleaser.com/) and releasing with a GitHub action ([example](https://github.com/goreleaser/goreleaser-action)), but maybe you have other ideas?

Also, I don't think we need a full docs site for now: the README plus examples work do, no?

**Comment by dankinder** [comment]
Even a manual tag initially would be great and encourage people to use it.

Thanks for this library! I'd like to use it but feel more comfortable with libraries that have hit v1.

**Comment by olivere** [comment]
@dankinder Understood. I use this library in production, if that helps. But I get your point.

**Comment by olivere** [comment]
I will focus on pushing a 1.0 release soon.
